index.html

```diff
@@ -4000,7 +4000,7 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
               </tr>
 
               <tr>
-                <td>0.005 cd/m<sup>2</sup></td>
+                <td>0.0005 cd/m<sup>2</sup></td>
```

Comment:
```suggestion
                <td>0.005 cd/m<sup>2</sup></td>
```

Oops.
I was able to resolve the conflicts and make this pull request clean.
However, I missed this line.
This should be 0.005, not 0.0005.